cedar-policy-core/src/est/err.rs

```diff
@@ -20,41 +20,41 @@ use crate::parser::unescape;
 use smol_str::SmolStr;
 use thiserror::Error;
 
-/// Errors arising while converting EST to AST
+/// Errors arising while converting a policy from its JSON representation (aka EST) into an AST
 #[derive(Debug, Error)]
-pub enum EstToAstError {
+pub enum FromJsonError {
     /// Error while deserializing JSON
     #[error(transparent)]
     JsonDeserializationError(#[from] JsonDeserializationError),
     /// Tried to convert an EST representing a template to an AST representing a policy
-    #[error("tried to convert JSON representing a template to a Policy rather than Template: {0}")]
+    #[error("tried to convert JSON representing a template to a policy rather than template: {0}")]
```

Comment:
[nit] should we use "static" and "linked" policy?

Reply:
Yes. Let me do it.